homeassistant/components/yolink/binary_sensor.py

```diff
@@ -28,17 +34,35 @@ class YoLinkBinarySensorEntityDescription(BinarySensorEntityDescription):
     value: Callable[[str], bool | None] = lambda _: None
 
 
-SENSOR_DEVICE_TYPE = [ATTR_DEVICE_DOOR_SENSOR]
+SENSOR_DEVICE_TYPE = [
+    ATTR_DEVICE_DOOR_SENSOR,
+    ATTR_DEVICE_MOTION_SENSOR,
+    ATTR_DEVICE_LEAK_SENSOR,
+]
 
 SENSOR_TYPES: tuple[YoLinkBinarySensorEntityDescription, ...] = (
     YoLinkBinarySensorEntityDescription(
         key="state",
         icon="mdi:door",
         device_class=BinarySensorDeviceClass.DOOR,
-        name="state",
+        name="State",
         value=lambda value: value == "open",
         exists_fn=lambda device: device.device_type in [ATTR_DEVICE_DOOR_SENSOR],
     ),
+    YoLinkBinarySensorEntityDescription(
+        key="state",
```

Comment:
This shares the same key with https://github.com/home-assistant/core/blob/ee659dcc36d0725c82fae8a4636c01978a9033b2/homeassistant/components/yolink/binary_sensor.py#L45 is it guaranteed that `device.device_id` will be different?
else you get the same unique id for both sensors:
https://github.com/home-assistant/core/blob/ee659dcc36d0725c82fae8a4636c01978a9033b2/homeassistant/components/yolink/binary_sensor.py#L105

Reply:
Yes,Each sensor has its own unique device_id .